x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -91,24 +88,10 @@ type SecuritySolutionDeepLink = AppDeepLink & {
 };
 
 export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
```

Comment:
This is the file defining the old navigation inks. The rest of the files define the new navigation links